.readthedocs.yml

```diff
@@ -0,0 +1,36 @@
+# .readthedocs.yaml
+---
+# Read the Docs configuration file
+# See https://docs.readthedocs.io/en/stable/config-file/v2.html for details
+
+# Required
+version: 2
+
+# Set the version of Python and other tools you might need
+build:
+  os: ubuntu-20.04
+  tools:
+    python: "3.9"
+    # You can also specify other tool versions:
+    # nodejs: "16"
+    # rust: "1.55"
+    # golang: "1.17"
+
+sphinx:
+  fail_on_warning: true
+
+# Build documentation in the docs/ directory with Sphinx
+# sphinx:
+#    configuration: docs/conf.py
+
+# If using Sphinx, optionally build your docs in additional formats such as PDF
+# formats:
+#    - pdf
+
+# Optionally declare the Python requirements required to build your docs
+python:
+  install:
+    - requirements: requirements.txt
```

Comment:
Reading https://docs.readthedocs.io/en/stable/config-file/v2.html#python-install, it should be possible to install  `ome_zarr`  and its dependencies without creating a new requirements txt file using the following syntax:

```
python:
  install:
    - method: pip
      path: .
```